samcli/local/docker/lambda_container.py

```diff
@@ -201,3 +217,29 @@ def _get_entry_point(runtime, debug_options=None):  # pylint: disable=too-many-b
             runtime=runtime,
             options=LambdaContainer._DEBUG_ENTRYPOINT_OPTIONS,
         )
+
+    @staticmethod
+    def _get_additional_volumes(host_volumes):
+        """
+        Generate a dictionary for mounting additional volumes in container.
+        A remote directory is generated based on remote base path + host node folder name
+
+        :param Tuple(Path) host_volumes: Volumes to be mounted in a container.
+        :return dict: A dictionary containing volumes mapping between host and remote container
+        """
+
+        if not host_volumes:
+            return None
+
+        volume_map = {}
+        for host_volume in host_volumes:
+            host_base_names = os.path.basename(host_volume)
+
+            if not host_base_names:
+                continue
+
+            remote_volume_path = os.path.join(LambdaContainer._VOLUME_MOUNT_PATH, host_base_names)
```

Comment:
Would this work on Windows?

Reply:
Good catch. Thank you. Updated to use PosixPath here.